tests/foreman/api/test_templatesync.py

```diff
@@ -185,6 +185,44 @@ def test_import_filtered_templates_from_git_with_negate(self, module_org):
         )
         assert len(rtemplates) == 1
 
+    @pytest.mark.tier2
+    def test_import_template_with_puppet(self, parametrized_puppet_sat):
+        """Importing puppet templates with enabled/disabled puppet module
+
+        :Steps:
+            1. Have enabled(disabled) puppet module
+            2. Import template containing puppet
+            3. Check if template was imported
+
+        :expectedresults:
+            Template was (not) imported
+
+        :id: 9ab46883-a3a7-4d2c-9a79-3d574bfd2ad8
+
+        :parametrized: yes
+
+        :CaseImportance: Medium
+        """
+        prefix = gen_string('alpha')
+        api_template = parametrized_puppet_sat['sat'].api.Template()
+        filtered_imported_templates = api_template.imports(
+            data={
+                'repo': FOREMAN_TEMPLATE_IMPORT_URL,
+                'branch': 'automation',
+                'dirname': 'import/job_templates/',
+                'filter': 'jenkins Start OpenSCAP scans',
+                'force': True,
+                'prefix': prefix,
+            }
+        )
+        count = [
```

Comment:
I suggest renaming the variable for the sake of better readability something like `not_imported_count` or `count_non_imported` would be better. Non-blocking though...
```suggestion
        not_imported_count = [
```